.github/workflows/docs-test.yml

```diff
@@ -0,0 +1,38 @@
+name: Docs Test
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags:
+      - "**"
+    paths:
+      - "src/**"
+      - "docs/**"
+
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
```

Comment:
Is this correct? Seems like it should not be ignored...?

Reply:
Of course ... it's a typo. Fixed!